app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -29,47 +29,73 @@
-                            <SortButton @sortBy='department' />
+                            <div class="header-content">
+                                <strong>{{t 'institutions.dashboard.users_list.department'}}</strong>
+                                <SortButton @sortBy='department' />
+                            </div>
                         </th>
-                        <th local-class='nested-header text-center' colspan='2' width='128'>
-                            {{t 'institutions.dashboard.users_list.projects'}}
+                        <th>
+                            {{t 'institutions.dashboard.users_list.osf_link'}}
                         </th>
-                    </tr>
-                    <tr>
-                        <th local-class='nested-header text-center' data-test-header-public-projects>
-                            <img src='/assets/images/global/open-lock.svg'
-                                alt={{t 'institutions.dashboard.users_list.public_projects'}}
-                            >
+                        <th>
+                            <div class="header-content">
+                                <span>{{t 'institutions.dashboard.users_list.public_projects'}}</span>
+                                <SortButton @sortBy='public_projects' />
+                            </div>
+                        </th>
+                        <th style="padding-left: 0px;" >
+                            <div class="header-content">
+                                <span>{{t 'institutions.dashboard.users_list.private_projects'}}</span>
+                                <SortButton @sortBy='private_projects' />
+                            </div>
+                        </th>
+                        <th>
+                            <div class="header-content">
+                                {{t 'institutions.dashboard.users_list.public_registration_count'}}
+                                <SortButton @sortBy='public_registration_count' />
+                            </div>
                         </th>
-                        <th local-class='nested-header text-center' data-test-header-private-projects>
-                            <img src='/assets/images/global/closed-lock.svg'
-                                alt={{t 'institutions.dashboard.users_list.private_projects'}}
-                            >
+                        <th>
+                            <div class="header-content">
+                                {{t 'institutions.dashboard.users_list.private_registration_count'}}
+                                <SortButton @sortBy='embargoed_registration_count' />
+                            </div>
+                        </th>
+                        <th>
+                            <div class="header-content">
+                                {{t 'institutions.dashboard.users_list.published_preprint_count'}}
+                                <SortButton @sortBy='preprint' />
```

Comment:
Per the API contract, I think this should be `@sortBy='published_preprint_count'`